pkg/client/sippy/v1/types.go

```diff
@@ -0,0 +1,21 @@
+package v1
+
+import (
+	bugsv1 "github.com/openshift/sippy/pkg/apis/bugs/v1"
+	sippyv1 "github.com/openshift/sippy/pkg/apis/sippy/v1"
+)
+
+// releaseReport is returned from Sippy /json API endpoint and it contains map with release name and the actual report.
+// TODO: Fix the returned data and move this map to single structure
+type releaseReport map[string]*Report
+
+// Report contain a single release report returned from Sippy API.
+type Report struct {
```

Comment:
I don't like having a new type unless it's a strict subset of a different type we already have.

Reply:
@deads2k it is a wrapper for existing types

Reply:
I tend to agree with @deads2k, why can't the server and client share a common type for the resource being provided/consumed via the api endpoint?